### DISCUSSION TITLE
How to force webview to be opened NOT in preview mode?

### DISCUSSION THREAD

**Discussion by dschonholtz** [microsoft/vscode-discussions#1110] [Extension Development QnA] [answered]
Hi there, 
I am working on a vscode extension, when we reveal it, it seems to consistently open with the webview italicized. If a user clicks on a new file then the webview is destroyed!
I have seen some stuff about webviewId.focus as a command, but we have tested that to no avail.
We also have combed through the webView API and don't see a way to handle this.

Any help would be extremely helpful. We just want to always be able to open our webview as a permanent panel.

**Comment by alefragnani** [comment, marked as answer]
Hi @dschonholtz ,

Are you using the `vscode.window.createWebviewPanel` API? If yes, which are the params?

I'm using this API for a while, and I don't remember it opening the webview in preview mode.

Hope this helps
